search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/TenantFilterTransformer.java

```diff
@@ -31,16 +29,16 @@ public TenantFilterTransformer(final IndexDescriptor indexDescriptor) {
   @Override
   public SearchQuery toSearchQuery(final TenantFilter filter) {
     return and(
-        term(TenantIndex.JOIN, IdentityJoinRelationshipType.TENANT.getType()),
         filter.key() == null ? null : term(KEY, filter.key()),
         filter.tenantId() == null ? null : term(TENANT_ID, filter.tenantId()),
         filter.name() == null ? null : term(NAME, filter.name()),
-        filter.memberKeys() == null
```

Comment:
`memberKeys` is still on TenantFilter. Do we drop it here intentionally?

Reply:
I dropped it intentionally as we were shifting through the resources anyway but good point, I'll reinstate it for now